tests/link/jax/test_nlinalg.py

```diff
@@ -96,7 +96,7 @@ def test_jax_basic_multiout_omni():
 
 @pytest.mark.xfail(
     version_parse(jax.__version__) >= version_parse("0.2.12"),
-    reason="Omnistaging cannot be disabled",
+    reason="`dot` -> `Gemv` optimization is incompatible with JAX",
```

Comment:
I don't see why this test should fail in the first place

Reply:
I double-checked the test case. It passed now so I remove the expected failure.